src/citrine/resources/descriptors.py

```diff
@@ -0,0 +1,40 @@
+from typing import List
+from uuid import UUID
+
+from citrine._session import Session
+from citrine.informatics.descriptors import Descriptor
+from citrine.informatics.predictors import Predictor
+
+
+# Not a full Collection since CRUD operations are not valid for Descriptors
+class DescriptorMethods:
+    """[ALPHA] A set of methods returning descriptors which require connection to the backend."""
+
+    def __init__(self, project_id: UUID, session: Session):
+        self.project_id = project_id
+        self.session: Session = session
+
+    def from_predictor_responses(self, predictor: Predictor,
+                                 inputs: List[Descriptor]) -> List[Descriptor]:
+        """
+        [ALPHA] Get responses for a predictor, given a set of inputs.
+
+        Parameters
+        ----------
+        predictor The predictor whose available responses are to be computed.
+        inputs The inputs to the predictor
+
+        Returns
+        -------
+        List[Descriptor]
+            The computable responses of the predictor given the provided inputs.
+
+        """
+        response = self.session.post_resource(
+            path='/projects/{}/material-descriptors/predictor-responses'.format(self.project_id),
```

Comment:
Does it actually make sense to have to do this in the context of a project?

Reply:
We're not using any of the project resources of permissions, so no, the api route does not need to be on the `/projects/` path.

That said, we should decouple what makes sense for API design from what makes sense to show the user. The latter decision should involve product. We did the solution design exercise (I'm sorry for not directing you to this sooner) and the results are here: https://github.com/CitrineInformatics/ai-engine-documentation/blob/master/docs/predictors.md#retrieving-predictor-responses

Our plan was to expose the method as `predictor.get_responses(inputs)`. I realize now that's not so easy because a given `predictor` doesn't necessarily have `Session` information. But we should still try and make the user interface as readable as possible.

Reply:
I think I like `descriptors.from_predictor_responses(predictor, inputs)`. It seems like it will be easy for users to grasp.

Reply:
What if its a graph predictor that references other predictors by UID? I think we do want to keep just about everything scoped in a project because, even if it doesn't require access checks now it might in the future.

Reply:
Are you suggesting we handle graph predictors with references now, or that we might in the future? I agree with the latter, but to be clear I don't think the backend supports the former right now (but @asantas93 will know for sure)

Reply:
This type signature doesn't exclude them.  Graph predictors do support references already.  The backend [seems to](https://github.com/CitrineInformatics/platform-backend/pull/1180/files#diff-176be3fadd0613e04391b60237f80672R64).